packages/neuvector/generated-changes/patch/values.yaml.patch

```diff
@@ -0,0 +1,58 @@
+--- charts-original/values.yaml
++++ charts/values.yaml
+@@ -4,8 +4,8 @@
+ 
+ openshift: false
+ 
+-registry: registry.neuvector.com
+-tag: 4.4.3
++registry: docker.io
```

Comment:
Need to use system_default_registry: https://github.com/rancher/charts/blob/b55e6ec01907964723942fdbd9f63b9afef7a1ed/packages/rancher-cis-benchmark/charts/templates/_helpers.tpl#L6-L12